releases/cosmwasm/2025-02-xrplevm-GMP-v6.4.0.md

```diff
@@ -0,0 +1,351 @@
+# Ripple EVM Sidechain GMP v6.4.0
+
+|                | **Owner**                                                                   |
+| -------------- | --------------------------------------------------------------------------- |
+| **Created By** | @blockchainguyy <ayush@interoplabs.io>                                      |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io>, @talalashraf <talal@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [Amplifier Releases](https://github.com/axelarnetwork/axelar-amplifier/releases)
+- [VotingVerifier v1.1.0](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/voting-verifier-v1.1.0)
+- [Gateway v1.1.1](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/gateway-v1.1.1)
+- [MultisigProver v1.1.1](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/multisig-prover-v1.1.1)
+
+## Background
+
+These are the instructions for deploying Amplifier contracts for Ripple EVM connection.
+
+### Pre-requisites
+
+Predict the [External Gateway](../evm/2025-02-xrplevm-GMP-v6.4.0.md) address, as `VotingVerifier` needs the `sourceGatewayAddress` which is the External Gateway address.
+
+| Network              | `minimumRotationDelay` | `deploymentType` | `deployer`                                   |
+| -------------------- | ---------------------- | ---------------- | -------------------------------------------- |
+| **Devnet-amplifier** | `0`                    | `create3`        | `0xba76c6980428A0b10CFC5d8ccb61949677A61233` |
+| **Stagenet**         | `300`                  | `create3`        | `0xba76c6980428A0b10CFC5d8ccb61949677A61233` |
```

Comment:
To repeat my comment from above, our staging environment on testnet can be considered `stagenet` here. So yes, apporpriate to use 0xba76c6980428A0b10CFC5d8ccb61949677A61233